docs/src/components/SearchInput/primitives/StyledDropdown.ts

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const StyledDropdown = styled.ul`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  top: 60px;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusLarge};
+  padding: 1em 0;
+  background: #fff;
+  min-width: 600px;
+  width: 100%;
+  z-index: 10;
+  box-shadow: ${({ theme }) => theme.orbit.boxShadowAction};
+`;
+
+export const StyledDropdownItem = styled.li`
+  padding: 0.5em 1em;
+  color: ${({ theme }) => theme.orbit.paletteProductNormal};
+  &:hover {
+    color: ${({ theme }) => theme.orbit.paletteProductNormalHover};
+  }
+  &:active {
+    color: ${({ theme }) => theme.orbit.paletteProductNormalActive};
+  }
+  &:nth-child(even) {
+    background: ${({ theme }) => theme.orbit.paletteCloudNormal};
+  }
+  outline: none;
+  > p {
+    font-size: calc(1em - 2px);
+    color: ${({ theme }) => theme.orbit.colorTextPrimary};
+  }
+`;
```

Comment:
Lots of interpolated functions here, I recommend saving some effort (and performance) with `css`:

```js
const StyledDropdownItem = styled.li`
  ${({ theme }) => css`
    ...
  `}
`
```

then inside you can put all the styles and interpolate `theme` values directly, instead of functions.

Reply:
i know about that, thanks